dhis-2/dhis-web-api/src/test/java/org/hisp/dhis/webapi/controller/tracker/export/event/EventRequestParamsMapperTest.java

```diff
@@ -452,4 +457,61 @@ void shouldThrowWhenOrderParameterContainsRepeatedOrderComponents() {
         () -> assertContains("enrolledAt", exception.getMessage()),
         () -> assertContains("zGlzbfreTOH", exception.getMessage()));
   }
+
+  @Test
+  void shouldFailWhenOrgUnitSuppliedAndOrgUnitModeAccessible() {
```

Comment:
Do we need all these tests in all the controllers even though they call a tested function from RequestParamUtils that is tested in RequestParamUtilsTest? Imagine how many tests would fail for the same reason if someone would add a bug in the validation function 😅

Reply:
Right, but for now I'll leave them in the old API and in the events mapper of the new one, until we refactor the events controller to accept a semi colon separated value as the input of `orgUnit.`